packages/schematics/angular/component/files/__name@dasherize@if-flat__/__name@dasherize__.__type@dasherize__.spec.ts.template

```diff
@@ -8,7 +8,7 @@ describe('<%= classify(name) %><%= classify(type) %>', () => {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      <%= standalone ? 'imports' : 'declarations' %>: [<%= classify(name) %><%= classify(type) %>]
+      <%= standalone ? '' : 'declarations' %>: [<%= classify(name) %><%= classify(type) %>]
```

Comment:
This does not seem right, when standalone you need to remove the entire object liternal

Reply:
The `configureTestingModule` method requires an object literal even if it is empty.

Reply:
In the case, you need an remove the imports and it's value not just the key. As right now it's not valid.